spec/features/admin/edit_tutorial_spec.rb

```diff
@@ -4,7 +4,7 @@
   let(:tutorial) { create(:tutorial) }
   let(:admin)    { create(:admin) }
 
-  scenario "by adding a video", :js do
+  scenario "by adding a video", :js, :vcr do
```

Comment:
I need to get on the VCR gem... did you come across any tutorials or vids that you can recommend?